.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -45,10 +45,6 @@ jobs:
         export GIT_BRANCH=$BUILD_SOURCEBRANCHNAME
         .scripts/run_docker_build.sh
     displayName: Run docker build
-  - script: |
-        rm -rf /opt/ghc
-        df -h
-    displayName: Manage disk space (rm /opt/ghc)
```

Comment:
Would it be possible to move this before the Docker run step? Maybe that addresses the issue? @isuruf @beckermr, please correct me if I'm still not understanding things here.

Reply:
The env block below needs to be attached to the docker run step so that the env vars are passed in. You'd be best off submitting a patch to smithy.

Reply:
Ok, so doing this before the Docker image part should be fine.

Yeah no objections submitting to smithy, but it sounds like we still need this in the interim as noted in the OP here ( https://github.com/conda-forge/faiss-split-feedstock/pull/5 ). We also should make sure it is working correctly before adding to smithy.

Reply:
i'd consult the azure docs on their CI configuration. The change you put in was causing these env vars to be injected into the wrong part of the job. The `env: ...` is part of the list element that has the docker run script.

Reply:
Since master branch passes with the deleting as the last step, it means this step is not needed.

Reply:
IOW would PR ( https://github.com/conda-forge/faiss-split-feedstock/pull/10 ) do what we want? If so, happy to add to conda-smithy as well.

Reply:
I just committed the proper modification to this step if you want it.

Reply:
This step is not needed as the commits work fine. Can we just rerender and remove this unnecessary hack?

Reply:
> i'd consult the azure docs on their CI configuration. The change you put in was causing these env vars to be injected into the wrong part of the job. The `env: ...` is part of the list element that has the docker run script.

I didn't add it ;) But yeah that makes sense. Thanks for clarifying :)

> Since master branch passes with the deleting as the last step, it means this step is not needed.

It may be that is flaky. IDK would ask @h-vetinari who knows more.

> I just committed the proper modification to this step if you want it.

Thanks! 😄

Reply:
> @jakirkham: It may be that is flaky. IDK would ask @h-vetinari who knows more.

Sometime between testing in #1 and merging to master we crossed the threshold to run out of disk space (whether through being very close to it before, and/or changes in the azure setup). I tried to rerun in #5 and it turned out _not_ to be flaky - it would fail on every run. I then got pointed to conda-forge/omniscidb-feedstock#5 (and https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/105), whereupon I decided to reuse the hack to get the packages built, before the more stable infra update comes through.

Thanks everyone for trying to get this sorted!